templates/Components/_header.html.twig

```diff
@@ -0,0 +1,5 @@
+<div class="header">
```

Comment:
I think `<header>` is better choice than `<div>` :)

Reply:
thanks i'll make the change